src/octoprint/plugins/eventmanager/__init__.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+from flask_babel import gettext
+
+import octoprint.events
+import octoprint.plugin
+from octoprint.access import ADMIN_GROUP
+
+
+class EventManagerPlugin(
+    octoprint.plugin.SettingsPlugin,
+    octoprint.plugin.AssetPlugin,
+    octoprint.plugin.TemplatePlugin,
+):
+    def on_settings_load(self):
+        my_settings = {
+            "availableEvents": octoprint.events.all_events(),
+            "subscriptions": [],
+        }
+        events = self._settings.global_get(["events"])
+        if events:
+            my_settings["subscriptions"] = events["subscriptions"]
```

Comment:
```suggestion
            my_settings["subscriptions"] = events.get("subscriptions", [])
```

(people might have garbage in their `config.yaml`)